benchalerts/benchalerts/pipeline_steps/conbench.py

```diff
@@ -11,17 +12,46 @@
 ConbenchClient = LegacyConbenchClient
 
 
-class GetConbenchZComparisonStep(AlertPipelineStep):
-    """An ``AlertPipeline`` step to get information from Conbench comparing the runs on
-    a contender commit to their baselines, using a z-score threshold. This is always the
-    first step of the pipeline.
+CONBENCH_ENV_VAR_HELP = """
+
+    Notes
+    -----
+    Environment variables
+    ~~~~~~~~~~~~~~~~~~~~~
+    ``CONBENCH_URL``
+        The URL of the Conbench server. Only required if ``conbench_client`` is not
+        provided.
+    ``CONBENCH_EMAIL``
+        The email to use for Conbench login. Only required if ``conbench_client`` is not
+        provided and the server is private.
+    ``CONBENCH_PASSWORD``
+        The password to use for Conbench login. Only required if ``conbench_client`` is
+        not provided and the server is private.
+"""
+
+
+class BaselineRunCandidates(Enum):
+    """Types of baselines available from `/api/runs/{run_id}` in the
+    `candidate_baseline_runs` field.
+    """
+
+    fork_point = "fork_point"
+    head_of_default = "head_of_default"
```

Comment:
I was going by [staging redoc](https://cb-staging.voltrondata.run/api/redoc#tag/Runs/paths/~1api~1runs~1%7Brun_id%7D~1/get) which still says `head_of_default`? I'm confused

Reply:
Oh no I forgot to change it there! Will do in a followup

Reply:
I could change it here if it's just docs, searching for `head_of_default` it looks like it's in a fair number of places still...

Reply:
Wait, mostly `test_history.py`. Should I just fix it?

Reply:
I opened https://github.com/conbench/conbench/pull/1179 already.

I'm leaving the phrase `head_of_default` in `test_history.py` on purpose, because it's a slightly different thing. It's not talking about a candidate baseline run, just a strategy for pytesting z-scores.